cumulusci/tasks/bulkdata/update_data.py

```diff
@@ -224,3 +225,16 @@ def _object_description(self, obj):
             return f'{obj} objects matching "{self.options["where"]}"'
         else:
             return f"all {obj} objects"
+
+
+class CumulusCIUpdatesApplication(SnowfakeryApplication):
+    """Takes over Snowfakery logging so CumulusCI can control it"""
+
+    def __init__(self, logger) -> None:
+        self.logger = logger
+        super().__init__()
+
+    def echo(self, message, *args, **kwargs):
+        # skip CSV creation messages
+        if not message.startswith("Created "):
```

Comment:
This seems a bit hacky and prone to unintentionally suppressing messages about other things being created. Wouldn't it make more sense to pass a flag to snowfakery which tells it not to emit the message in the first place?

Reply:
@davisagli Either way the parent app and Snowfakery would need some way of communicating which messages to send or not send. I did tighten up the test so that it's essentially impossible for it to match anything I don't want it to.